sidecars/ruby/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+#   Lining Pan
+
+FROM ruby:2.6.6-stretch
```

Comment:
if you want to maintain all the arch's provided by official base image you need to upgrade the base image to 2.7.2-buster as it appears the stretch images have been deprecated

Reply:
Thanks for the insight, I'll handle ruby in a separate PR.